app/views/partials/droits-eligibles-list.html

```diff
@@ -1,9 +1,8 @@
-<accordion close-others="false">
-  <accordion-group is-open="droit.open"
-               class="resultats-accordion"
-               ng-repeat="(droitId, droit) in list|orderObjectBy:'label'"
-               ng-class="{'resultats-accordion-needs-n-2': shouldDisplayYM2Warning(droit), 'panel-opened': droit.open}">
-    <accordion-heading>
+<uib-accordion close-others="false">
+  <uib-accordion-group is-open="droit.open"
+     class="resultats-accordion {{ getAccordionClass(droit) }}"
```

Comment:
Could this be made more readable by using an object `ng-class` syntax, like:

```
{ resultats-accordion: true,
  resultats-accordion-needs-n-2: shouldDisplayYM2Warning(),
  panel-opened: droit.open }
```

?